### ISSUE TITLE
Breakpoints do not get hit on nightly Rust 1.60 MSVC

### ISSUE THREAD

**Issue by chyyran** [intellij-rust/intellij-rust#8526]
## Environment

* **IntelliJ Rust plugin version:** 0.4.166.4422-213-nightly
* **Rust toolchain version:** nightly-x86_64-pc-windows-msvc rustc 1.60.0-nightly (f624427f8 2022-02-06)
* **IDE name and version:** CLion 2021.3.3
* **Operating system:** Windows 10 19044.1466

## Problem description
Breakpoints do not get hit when building against nightly Rust, and become cross/greyed out. Breakpoints are fine on stable Rust.

Cargo run configuration is set to `[default]`. I've also tried `nightly` but it does not work either.
![image](https://user-images.githubusercontent.com/1000503/152874853-c1796ae3-097b-4218-981f-d7ecd1a89435.png)

## Steps to reproduce
1. Create a simple rust program

```rust
fn main() {
    println!("Hello, world!");
    loop{}
}
```

2. Set toolchain to nightly
```
> rustup default nightly
```

3. Clean directory

```
> cargo clean
```

4. Set a breakpoint on any line and try to debug in CLion

5. Breakpoints gray out.
![image](https://user-images.githubusercontent.com/1000503/152874644-cdb70801-0ebe-424a-99dc-bbbd01f49890.png)

**Comment by rmccrystal** [comment]
Having the same issue this is annoynig as fuck, other issues attribute this to the wrong lldb version (fixed in eap) but that didn't fix it for me. I guess I just need to downgrade rust

**Comment by chyyran** [comment]
Just tested on 2022.1 EAP and can confirm that the issue persists.

**Comment by rmccrystal** [comment]
Solved it by downgrading to `1.58.0` or `nightly-2022-01-01`

**Comment by ortem** [comment]
@chyyran @rmccrystal Thank you for reporting, there is indeed a problem with breakpoints when using 1.60.0 nightly MSVC Rust toolchain. Downgrading the Rust toolchain to stable 1.58.0 or even beta 1.59.0 helps. We are currently investigating this problem.

**Comment by ortem** [comment]
@chyyran @rmccrystal Please check out the latest [CLion EAP](https://www.jetbrains.com/clion/nextversion/) build. With this build, breakpoints should work fine.

**Comment by chyyran** [comment]
![image](https://user-images.githubusercontent.com/1000503/156903206-7e1d94c9-066f-4391-8f83-6c8ed2c62d6b.png)

Working for me on rustc 1.61.0-nightly (9fcbc3205 2022-03-04), CLion 2022.1 EAP 221.4906.7

**Comment by chyyran** [comment]
I updated to the nightly plugin on 2021.3.3 and it doesn't seem to work. Will this fix be backported to 2021.3.3 or will we have to wait for 2022.1 to release (and use EAP in the meantime?)

**Comment by Undin** [comment]
> I updated to the nightly plugin on 2021.3.3 and it doesn't seem to work

It's expected because the fix was made in lldb itself, not in the Rust plugin.

> Will this fix be backported to 2021.3.3 or will we have to wait for 2022.1 to release (and use EAP in the meantime?)

@ortem could you ask CLion team about it, please?

**Comment by ortem** [comment]
@chyyran Thank you for confirming that EAP 221.4906.7 fixes the problem. The fix will be backported to CLion 2021.3.3 as well.
JFYI: [JetBrains Toolbox App](https://www.jetbrains.com/toolbox-app/faq/) might help you to manage your IDEs and to quickly switch between EAP and stable versions.

**Comment by neonaot** [comment]
Breakpoints work fine in 2021.3.4 👍
